### DISCUSSION TITLE
Klaviyo module is not accessible in the Saleor

### DISCUSSION THREAD

**Discussion by palash-cj** [saleor/apps#336] [Q&A] [answered]
Hi There!

I want to integrate SMS services in Saleor using Klaviyo. For this I am creating a plugin where I am triggering the SMS event. To do so, Klaviyo module is required. I installed it with command - 'pip install klaviyo'. 
![Screenshot (67)](https://user-images.githubusercontent.com/69688293/227435484-6193390c-1555-4642-92d1-7154c0825b2a.png)

But even after installing it, getting the error like 'No module named klaviyo' from running the command -'docker compose up'
![Screenshot (68)](https://user-images.githubusercontent.com/69688293/227435928-c360d708-59d2-46c0-abd0-60bd01c334b9.png)

![Screenshot (69)](https://user-images.githubusercontent.com/69688293/227436465-d35514db-d492-4cfb-9c4d-34ff5d9009fb.png)

This is my current code where am getting the error
![Screenshot (71)](https://user-images.githubusercontent.com/69688293/227436391-21fc3583-986d-4bc5-a036-1bf58351776c.png)

**Comment by karolkielecki** [comment, marked as answer]
Hi @palash-cj, we don't recommend using plugins. Plugins will be deprecated soon. If you want to extend Saleor, you need to develop an app. More details here: https://docs.saleor.io/docs/3.x/developer/extending/apps/key-concepts

We've recently released [Saleor App Store](https://github.com/saleor/apps) together with a [Klaviyo app](https://github.com/saleor/apps/tree/main/apps/klaviyo).

**Reply by palash-cj** [reply]
Hi @karolkielecki , I already tried the way what you suggested. I installed the klaviyo app with manifest url but this app did not work as I wish. 

Now, I am creating a plugin to customize the functionalities as I want using klaviyo module. But facing the error.

**Reply by patrys** [reply]
@palash-cj, could you open a discussion in the `apps` repository, describing what you did not like about the Klaviyo app?
